src/richchk/mpq/stormlib/stormlib_finder.py

```diff
@@ -29,7 +30,7 @@ def find_stormlib(
         else:
             cls._LOG.warning(
                 "No path to a StormLib DLL was provided, "
-                "attempting to use precompiled DLL for Windows and macOS.  "
+                "attempting to use precompiled DLL for Windows, macOS and Linux. "
```

Comment:
Will this work for every flavor of 64-bit Linux?  Ubuntu, centos, Debian, etc.?  

How was this DLL compiled?  Did you compile it?  Or where did it come from?

Reply:
Yes, it should work. 

The library was compiled by myself. I am using Debian 12.

Reply:
Thanks for letting me know!  Only thing I would add, could you provide the instructions you did to compile the StormLib DLL on your system?  Either in the PR request or a link to something, etc.  Wanting to make sure it can be repeated.  E.g. for macOS StormLib, Homebrew builds it: https://formulae.brew.sh/formula/stormlib

Reply:
The instructions are described in the StormLib repository: https://github.com/ladislav-zezula/StormLib?tab=readme-ov-file#linux. One thing I added the `BUILD_SHARED_LIBS=True` flag to cmake.

```bash
git clone https://github.com/ladislav-zezula/StormLib.git
mkdir StormLib/build && cd StormLib/build
cmake ../CMakeLists.txt -D BUILD_SHARED_LIBS=True
make
```
and in the build folder will be compiled library `libstorm.so.9.22.0`